src/ray/raylet/worker_pool.cc

```diff
@@ -329,6 +329,9 @@ Process WorkerPool::StartWorkerProcess(
   }
 
   ProcessEnvironment env;
+  if (RayConfig::instance().enable_multi_tenancy()) {
+    env.emplace("RAY_JOB_ID", job_id.Hex());
```

Comment:
Can we use a constant for this?

Reply:
Also, please explain in a comment why we are doing this (since passing values through a job id are not consistent with other behaviors which pass values through a cli commands)

Reply:
done.